org.kde.okular.json

```diff
@@ -74,15 +74,30 @@
             "name": "unarchiver",
             "buildsystem": "simple",
             "build-commands": [
-                "sed -s -i 's#/usr/include/GNUstep#/app/include#g' */Makefile.linux && cd XADMaster && make -j $FLATPAK_BUILDER_N_JOBS -f Makefile.linux",
+                "sed -s -i 's#/usr/include/GNUstep#/app/include#g' */Makefile.linux",
```

Comment:
Will likely need  `cd XADMaster` like before here.

Reply:
make is run with option '-C' to that directory

Reply:
Ah, good point.